llvm/test/DebugInfo/CAS/AArch64/debug_unopt.ll

```diff
@@ -2,6 +2,7 @@
 ; RUN: llvm-cas-dump --cas=%t/cas --casid-file %t/debug_unopt.id | FileCheck %s
 ; CHECK: mc:debug_string_section llvmcas://
 ; CHECK-NEXT: mc:debug_string llvmcas://
+; CHECK-NEXT: mc:padding      llvmcas://
```

Comment:
In my commit refactoring, I didn't realize this change should be part of the previous commit